Formula/snowpack.rb

```diff
@@ -19,7 +19,19 @@ class Snowpack < Formula
 
   def install
     system "npm", "install", *Language::Node.std_npm_install_args(libexec)
-    bin.install_symlink Dir["#{libexec}/bin/*"]
+    bin.install_symlink Dir[libexec/"bin/*"]
+
+    # Remove incompatible pre-built binaries
+    os = "darwin"
+    on_linux { os = "linux" }
+    arch = Hardware::CPU.arm? ? "arm64" : "x64"
+    libexec.glob("lib/node_modules/snowpack/node_modules/{bufferutil,utf-8-validate}/prebuilds/*")
+           .each { |dir| dir.rmtree if dir.basename.to_s != "#{os}-#{arch}" }
+    # `rollup` < 2.38.3 uses x86_64-specific `fsevents`. Can remove when `rollup` is updated.
+    (libexec/"lib/node_modules/snowpack/node_modules/rollup/node_modules/fsevents").rmtree if Hardware::CPU.arm?
```

Comment:
There are a few different style alternatives to perform the deletion. Can be modified if any preference.

Reply:
we can also revisit this after `bufferutil`/`utf-8-validate` doing the universal binary.

Reply:
I just need to link without my hit.